From 6e0cc765b749a124974e38471a071f40b55e3e83 Mon Sep 17 00:00:00 2001
From: Hai Pham <hai.pham.ud@renesas.com>
Date: Mon, 26 May 2025 21:36:37 +0700
Subject: [PATCH 44/52] configs: ironhide: Enable Baremetal clk and module
 controllers

Enable baremetal clk and module controllers as tentative measures, until
features supported via SCP-FW.

Signed-off-by: Hai Pham <hai.pham.ud@renesas.com>
---
 configs/r8a78000_ironhide_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/r8a78000_ironhide_defconfig b/configs/r8a78000_ironhide_defconfig
index b28e6e7455c..111128d7f1b 100644
--- a/configs/r8a78000_ironhide_defconfig
+++ b/configs/r8a78000_ironhide_defconfig
@@ -20,3 +20,4 @@ CONFIG_SYS_CBSIZE=2048
 CONFIG_CMD_MEMTEST=y
 CONFIG_SYS_ALT_MEMTEST=y
 CONFIG_BAUDRATE=1843200
+CONFIG_BAREMETAL_CLK_MDL_CTRL=y
-- 
2.34.1

